.kitchen.cloud.yml

```diff
@@ -78,18 +74,21 @@ suites:
       - recipe[osl-openstack::default]
   - name: controller
     run_list:
+      - role[openstack_tk]
       - recipe[osl-openstack::ops_database]
       - recipe[osl-openstack::controller]
       - recipe[openstack-integration-test::setup]
-      - recipe[scl]
+      #- recipe[scl]
```

Comment:
This pull request is still a work-in-progress. I will likely remove it in the final form.